MAINTAINERS.md

```diff
@@ -0,0 +1,144 @@
+# SPIRE Maintainership Guidelines and Processes
+This document captures the values, guidelines, and processes that the SPIRE project and its maintainers adhere to. All SPIRE maintainers, in their independent and individual capacity, agree to uphold and abide by the text contained herein.
+
+This process can be changed, either permanently or as a one-time exception, through an 80% supermajority maintainer vote.
+
+For a list of active SPIRE maintainers, please see the [CODEOWNERS](CODEOWNERS) file.
+
+## General Governance
+The SPIRE project abides by the same [governance procedures][1] as the SPIFFE project, and ultimately reports to the SPIFFE TSC the same way that the SPIFFE project and associated maintainers do. While the SPIFFE TSC has the ultimate say, in practice they are only engaged upon serious maintainer disagreement. To say that this would be unprecedented is an understatement.
+
+TSC members do not track day-to-day activity in the SPIFFE/SPIRE projects, and this should be considered when deciding to raise issues to them.
```

Comment:
```suggestion
TSC members do not track day-to-day activity in the SPIFFE/SPIRE projects, and this should be considered when deciding to raise issues to them. While the SPIFFE TSC has the ultimate say, in practice they are only engaged upon serious maintainer disagreement. To say that this would be unprecedented is an understatement.
```